Assets/Mirror/Tests/Editor/MessageTests.cs

```diff
@@ -227,6 +227,22 @@ public void ReadyMessageTest()
             fresh.Deserialize(new NetworkReader(writerData));
         }
 
+        [Test]
+        public void AddPlayerMessageTest()
+        {
+            // try setting value with constructor
+            AddPlayerMessage message = new AddPlayerMessage();
+
+            // serialize
+            NetworkWriter writer = new NetworkWriter();
+            message.Serialize(writer);
+            byte[] writerData = writer.ToArray();
+
+            // deserialize the same data - do we get the same result?
+            AddPlayerMessage fresh = new AddPlayerMessage();
+            fresh.Deserialize(new NetworkReader(writerData));
+        }
```

Comment:
where are the asserts?  

Every test should have asserts,  because I can easily pass this test by just keeping Deserialize empty

Reply:
if the below is correct I will go through the rest of the class and fix the others like it.